frontend/lib/resolution/scope-queries.cpp

```diff
@@ -1106,13 +1156,15 @@ bool LookupHelper::doLookupInReceiverScopes(
   // using receiverScopes->isReceiverApplicable.
   int endSize = result.numIds();
   int cur = startParentsSize;
+  bool nonFunctions = false;
   for (int i = startParentsSize; i < endSize; i++) {
     IdAndFlags& idv = result.idAndFlags(i);
     if (receiverScopes->isReceiverApplicable(context, idv.id())) {
       // copy it to 'cur' unless it is already 'cur'
       if (cur != i) {
         result.idAndFlags(cur) = idv;
       }
+      nonFunctions |= !idv.isParenfulFunction();
```

Comment:
Should we also check against a method or field access here? As in `isFunctionLike`?